va_explorer/static/js/project.js

```diff
@@ -2,3 +2,10 @@
 $(function () {
   $('[data-toggle="tooltip"]').tooltip()
 })
+$('#hidden-check').change(function() {
+  var vis = this.checked ? 'none' : '';
+  $('.nan').css('display', vis);
+  $('td[class=""]').css('display',vis);
+})
+$('.nan').css('display', "none");
+$('td[class=""]').css('display',"none");
```

Comment:
See my comment below on `show.html`. Suggest tweaking the JavaScript slightly as follows:

```
$('#hidden-check').change(function() {
  $("tr[data-value]").toggle();
})
/* Hide rows where the form value is blank or "nan" by default */
$('tr[data-value="nan"]').css('display', "none");
$('tr[data-value=""]').css('display', "none");
```